### PR TITLE
updated expo android config docs

### PR DESCRIPTION
**Pull request by radixdev**
# Pull Request/Issue Resolution

#### Description of Change:
Added instructions on setting google-services.json file for expo

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [x] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the interal reviewing chart to tag the appropriate reviewer.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @timothy-kim or @KellieHawks for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@Timothy-Kim</td>
    <td>Data Infrastructure<br>Platform Infrastructure<br>Quality Infrastructure</td>
  </tr>
  <tr>
    <td>@kelliehawks</td>
    <td>Internal Tools<br>Product Partnerships<br>SDKs<br>SMS</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Channels<br>FIX<br>Intelligence<br>Reporting<br>SMB</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Email (Composition and Infrastructure)<br>Ingestion<br>Messaging</td>
  </tr>
</table>
</details>

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/react_native/push_notifications.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Integrating push notifications in React Native requires setting up each native p
 
 Set the `enableBrazeIosPush` and `enableFirebaseCloudMessaging` props to enable push for iOS and Android, respectively.
 
-For Android, you will also need to supply your `firebaseCloudMessagingSenderId`. See the [Android integration instructions]({{site.baseurl}}/developer_guide/platform_integration_guides/android/push_notifications/android/integration/standard_integration#step-4-set-your-firebase-credentials) for instructions on retrieving your sender ID.
+For Android, you will also need to supply your `firebaseCloudMessagingSenderId`. See the [Android integration instructions]({{site.baseurl}}/developer_guide/platform_integration_guides/android/push_notifications/android/integration/standard_integration#step-4-set-your-firebase-credentials) for instructions on retrieving your sender ID. You will additionally have to add your `google-services.json` file to `assets/google-services.json` in your application's expo folder. This file is required when setting `enableFirebaseCloudMessaging` to `true` in your configuration.
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/react_native/push_notifications.md:22]:
```suggestion
## Android Setup
Supply your `firebaseCloudMessagingSenderId`. See the [Android integration instructions]({{site.baseurl}}/developer_guide/platform_integration_guides/android/push_notifications/android/integration/standard_integration#step-4-set-your-firebase-credentials) on retrieving your sender ID. 

You will additionally have to add your `google-services.json` file to `assets/google-services.json` in your application's expo folder. This file is required when setting `enableFirebaseCloudMessaging: true` in your configuration.
```

File: _docs/_developer_guide/platform_integration_guides/react_native/push_notifications.md [hunk 2]
```diff
@@ -19,7 +19,10 @@ Integrating push notifications in React Native requires setting up each native p
 
 Set the `enableBrazeIosPush` and `enableFirebaseCloudMessaging` props to enable push for iOS and Android, respectively.
 
-For Android, you will also need to supply your `firebaseCloudMessagingSenderId`. See the [Android integration instructions]({{site.baseurl}}/developer_guide/platform_integration_guides/android/push_notifications/android/integration/standard_integration#step-4-set-your-firebase-credentials) for instructions on retrieving your sender ID.
+## Android Setup
+Supply your `firebaseCloudMessagingSenderId`. See the [Android integration instructions]({{site.baseurl}}/developer_guide/platform_integration_guides/android/push_notifications/android/integration/standard_integration#step-4-set-your-firebase-credentials) on retrieving your sender ID. 
+
+You will additionally have to add your `google-services.json` file to `assets/google-services.json` in your application's expo folder. This file is required when setting `enableFirebaseCloudMessaging: true` in your configuration.
```
Comment (wesleyorbin) [inline _docs/_developer_guide/platform_integration_guides/react_native/push_notifications.md:25]:
```suggestion

#### Step 1.1
Set the `firebaseCloudMessagingSenderId` config prop in your `app.json`. See the [Android integration instructions]({{site.baseurl}}/developer_guide/platform_integration_guides/android/push_notifications/android/integration/standard_integration#step-4-set-your-firebase-credentials) on retrieving your sender ID. 

#### Step 1.2
Add your `google-services.json` file to your application's`assets` folder. This file is required when setting `enableFirebaseCloudMessaging: true` in your configuration.
```

Comment (wesleyorbin) [reply _docs/_developer_guide/platform_integration_guides/react_native/push_notifications.md:25]: Thoughts on adding instructions on downloading the `google-services.json` file from Firebase? Or can we assume that people know how to do that?

Comment (radixdev) [reply _docs/_developer_guide/platform_integration_guides/react_native/push_notifications.md]: The "android integration instructions step 4" linked beforehand has those instructions.

File: _docs/_developer_guide/platform_integration_guides/react_native/push_notifications.md [hunk 3]
```diff
@@ -19,7 +19,10 @@ Integrating push notifications in React Native requires setting up each native p
 
 Set the `enableBrazeIosPush` and `enableFirebaseCloudMessaging` props to enable push for iOS and Android, respectively.
 
-For Android, you will also need to supply your `firebaseCloudMessagingSenderId`. See the [Android integration instructions]({{site.baseurl}}/developer_guide/platform_integration_guides/android/push_notifications/android/integration/standard_integration#step-4-set-your-firebase-credentials) for instructions on retrieving your sender ID.
+## Android Setup
+Supply your `firebaseCloudMessagingSenderId`. See the [Android integration instructions]({{site.baseurl}}/developer_guide/platform_integration_guides/android/push_notifications/android/integration/standard_integration#step-4-set-your-firebase-credentials) on retrieving your sender ID.
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/react_native/push_notifications.md]:
```suggestion
1. Supply your `firebaseCloudMessagingSenderId`. See the [Android integration instructions]({{site.baseurl}}/developer_guide/platform_integration_guides/android/push_notifications/android/integration/standard_integration#step-4-set-your-firebase-credentials) on retrieving your sender ID. 
```

File: _docs/_developer_guide/platform_integration_guides/react_native/push_notifications.md (hunk 2 above)
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/react_native/push_notifications.md]:
```suggestion
2. Add your `google-services.json` file to `assets/google-services.json` in your application's expo folder. This file is required when setting `enableFirebaseCloudMessaging: true` in your configuration.
```

**Review by davidbielik — APPROVED** [review]

File: _docs/_developer_guide/platform_integration_guides/react_native/push_notifications.md [hunk 4]
```diff
@@ -19,7 +19,13 @@ Integrating push notifications in React Native requires setting up each native p
 
 Set the `enableBrazeIosPush` and `enableFirebaseCloudMessaging` props to enable push for iOS and Android, respectively.
 
-For Android, you will also need to supply your `firebaseCloudMessagingSenderId`. See the [Android integration instructions]({{site.baseurl}}/developer_guide/platform_integration_guides/android/push_notifications/android/integration/standard_integration#step-4-set-your-firebase-credentials) for instructions on retrieving your sender ID.
+## Android Setup
```
Comment (KellieHawks) [inline _docs/_developer_guide/platform_integration_guides/react_native/push_notifications.md:22]:
```suggestion
### Android setup
```

**Review by KellieHawks — APPROVED** [review]

**Pull request merged**